src/screen/Task.jsx

```diff
@@ -1,15 +1,53 @@
 import React from "react";
-import { SafeAreaView, View } from "react-native";
+import { SafeAreaView, View, FlatList, StyleSheet } from "react-native";
 import { FirstHeader } from "../components/FirstHeader";
 import { CheckBox, CheckBoxs } from "../components/CheckBoxs";
 import { ClearButton } from "../components/ClearButton";
+import { colors, PRIMARY } from "../constants/colors";
 
-export const Task = () => (
-  <SafeAreaView style= {{backgroundColor: '#1F86FF'}}>
-    <FirstHeader />
-    <CheckBox />
-    <CheckBox />
-    <ClearButton />
-  </SafeAreaView>
+const DATA = [
+  {
+    id: 'pruebaid-123',
+    title: "Hacer el almuerzo",
+    subtitle: 'Empanadas de pollo'
+  },
+  {
+    id: 'pruebaid-12345',
+    title: "Hacer el checkbox",
+    subtitle: 'Incluirle titulo y subtitulos'
+  },
+  {
+    id: 'pruebaid-123456',
+    title: "Hacer la primera pantalla dinamica",
+    subtitle: 'No hardcodear'
+  },
+  {
+    id: 'pruebaid-1234567',
+    title: "Hacer la segunda pantalla",
+    subtitle: 'Boton de añadir'
+  },
+]
 
-);
+export const Task = () => {
+  const renderItem = ({item}) => (
+    <CheckBox title ={item.title} subtitle ={item.subtitle} />
+
+  );
```

Comment:
You can move this function outside the `Task` component, so you avoid re-creating the function every time a re-render occurs. Also, remember to apply prettier on those whitespaces.